postgrestools.jsonc

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://pgtools.dev/main/schema.json",
+  "vcs": {
+    "enabled": true,
+    "clientKind": "git",
+    "useIgnoreFile": true
+  },
+  "migrations": {
+    "migrationsDir": "apps/prairielearn/src/migrations",
+    "after": 20250000000000
+  },
+  "typecheck": {
+    "searchPath": ["public", "server*"]
+  },
+  "files": {
+    "ignore": ["*.pg"]
+  },
+  "linter": {
+    "rules": {
+      "all": false,
```

Comment:
`postgrestools` has an experimental copy of the `squawk` SQL linter. I have configured everything we want might to migrate to it, but not enabled it yet.